model/websocket_message.go

```diff
@@ -86,11 +86,12 @@ type WebSocketMessage interface {
 }
 
 type WebsocketBroadcast struct {
-	OmitUsers             map[string]bool `json:"omit_users"`    // broadcast is omitted for users listed here
-	UserId                string          `json:"user_id"`       // broadcast only occurs for this user
-	ChannelId             string          `json:"channel_id"`    // broadcast only occurs for users in this channel
-	TeamId                string          `json:"team_id"`       // broadcast only occurs for users in this team
-	ConnectionId          string          `json:"connection_id"` // broadcast only occurs for this connection
+	OmitUsers             map[string]bool `json:"omit_users"`         // broadcast is omitted for users listed here
+	UserId                string          `json:"user_id"`            // broadcast only occurs for this user
+	ChannelId             string          `json:"channel_id"`         // broadcast only occurs for users in this channel
+	TeamId                string          `json:"team_id"`            // broadcast only occurs for users in this team
+	ConnectionId          string          `json:"connection_id"`      // broadcast only occurs for this connection
+	OmitConnectionId      string          `json:"omit_connection_id"` // broadcast is omitted for this connection
```

Comment:
0/5 non-blocker:  I think this should be an array or map (like `OmitUsers`) so that we can omit more than one connection ID. I don't have a concrete use case in mind just yet, but I can see it being useful without making the code any more complicated.

Reply:
ATM we don't (can't?) have info about other connection ids.
Would it be meaningful to know another user's connection id, especially since one user could have many connections (depending on tabs/clients open at the same time)?

Reply:
Yeah, I can't think of an actual use case right now. This is not a blocker. We can change this whenever (if) required.